tensorflow_model_optimization/python/core/quantization/keras/graph_transformations/model_transformer.py

```diff
@@ -424,7 +424,7 @@ def _weight_name(name):
     Returns:
```

Comment:
Please update 'kernel:0' for 'dense_2/kernel:0' to one with two /'s to differentiate.

Reply:
@alanchiao  Sorry for butting in, could you please explain this review comment since we are making changes for clustering on this too?